pkg/apis/cartographer/v1alpha1/workload_source_printer.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2021 VMware, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
we should keep the apis packages independent of business logic about how output is rendered.

Reply:
Where would it be the best place to move this to? 
I was thinking about `pkg/cli-runtime/printer`